lib/src/tokens.dart

```diff
@@ -47,6 +47,9 @@ final Token $deferred = new KeywordToken(Keyword.DEFERRED, 0);
 /// The `/` token.
 final Token $divide = new Token(TokenType.SLASH, 0);
 
+/// The `do` keyword.
```

Comment:
can we consider dropping this lint? The zero-information comments are very noisy throughout